lib/js/_publish.js

```diff
@@ -9,6 +9,7 @@ const error = require('./_utils/_error');
 const extendPackage = require('./_utils/_extend-package').publish;
 const getPackages = require('./_utils/_get-packages');
 
+const npmIgnore = require('./_publish/_npm-ignore');
```

Comment:
to follow the existing naming conventions, this should be `_generate-npm-ignore`